pkgs/servers/dante/default.nix

```diff
@@ -10,7 +15,7 @@ stdenv.mkDerivation rec {
     sha256 = "0pbahkj43rx7rmv2x40mf5p3g3x9d6i2sz7pzglarf54w5ghd2j1";
   };
 
-  nativeBuildInputs = lib.optional stdenv.hostPlatform.isMips64 autoreconfHook;
+  nativeBuildInputs = [ autoreconfHook gnugrep gawk ];
```

Comment:
Weird this shouldn't be necessary
https://github.com/NixOS/nixpkgs/blob/8b2521bdae89126497fa6f139fb9d030d5d13184/pkgs/stdenv/generic/common-path.nix#L6-L7

Reply:
Yep, you're right.   74dc0681466c74aab517b706c2dfa10ec54d2e13